content/millennium/r4/scheduling/appointment.md

```diff
@@ -24,7 +24,7 @@ When updating an appointment, the resource provides the ability to change the [A
 * From Proposed to Booked, or Cancelled
 * From Booked to Arrived, Checked-In, or Cancelled
 * From Arrived to Checked-In, or Cancelled
-* From Checked-In to Cancelled
+* From Checked-In to Cancelled or Fulfilled
```

Comment:
let's make this
```
From Checked-In to Fulfilled, or Cancelled
```

to match the others.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/464/commits/3207a3c485e4126efcfe5a5dfe15cbc69b3cad34